lib/buy/onramper/onramper_buy_provider.dart

```diff
@@ -276,6 +279,10 @@ class OnRamperBuyProvider extends BuyProvider {
   String _getNormalizeCryptoCurrency(Currency currency) {
     if (currency is CryptoCurrency) {
       if (!mainCurrency.contains(currency)) {
+
+        final networkByCurrency = _getNetworkByCrypto(currency);
+        if (networkByCurrency != null) return '${currency.title}_$networkByCurrency';
```

Comment:
so this will be only the case of `CryptoCurrency.eos`?

can we re-write this function in a way that is more readable and friendly, even if it will duplicate some code

also doesn't this code mean that in the case of Monero it will return `XMR_MONERO` is this the expected value?
because for Bitcoin it will return just `BTC`

Reply:
Did some refactoring. Yes, for Monero the expected value is XMR_MONERO.